### ISSUE TITLE
Breaks on Arc

### ISSUE THREAD

**Issue by arpitdalal** [adyen-examples/adyen-testcards-extension#46]
For some reason, it doesn't even open up in Arc. Doesn't show any error messages anywhere. Just won't open like it does on Chrome. Adding a screen recording here showing other extensions like Wappalyzer opens correctly but this doesn't. Tried quitting and reopening Arc after installing the extension. Tried reinstalling the extension as well.

Any lead is helpful, I can help figure this out because I need it to work.

https://github.com/adyen-examples/adyen-testcards-extension/assets/61059807/4e99f015-9b01-481e-b8b0-4c202ae9dc58

**Comment by gcatanese** [comment]
Thank you @arpitdalal for reporting this. I have had a look and it seems that Arc doesn't support the `chrome.sidePanel` API. There is no a lot of documentation neither the source code of the extension so I cannot confirm it, I can only read some discussions on Reddit.

I have tried the Wappalyzer extension: it doesn't use the `chrome.sidePanel` but it opens instead a modal. Both approaches work fine, but we have decided to design the Adyen Test Cards extension using the `chrome.sidePanel` as it delivers a [better user experience](https://developer.chrome.com/blog/extension-side-panel-launch).
